Resources/public/js/views/services/ez-roleserversideviewservice.js

```diff
@@ -295,5 +299,19 @@ YUI.add('ez-roleserversideviewservice', function (Y) {
                 }
             });
         },
+
+        /**
+         * Refreshes the role view, showing the assignments tab.
+         *
+         * @method _refreshAssignmentsTab
+         * @protected
+         * @param {Object} data
+         */
+        _refreshAssignmentsTab: function (data) {
+            this.get('app').set('loading', true);
+            this.load(Y.bind(function () {
+                this.get('app').navigateTo('adminRole', {uri: 'pjax/role/view/' + data.roleId + '/ez-tabs-content'});
```

Comment:
this.load basically does an AJAX request (the same one as the one needed to generate the page the application is displaying) then once it's done you tell the application to navigate to the 'adminRole' route which will also request the URI `'pjax/role/view/' + data.roleId + '/ez-tabs-content'`.